### PR TITLE
Add activate app test

### PR DESCRIPTION
**Pull request by akinsolb**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [X] Have you proposed a file change/ PR with appium to update documentation? 
#### This can be done by navigating to the documentation section on http://appium.io selecting the appropriate command/endpoint and clicking the 'Edit this doc' link to update the C# example
https://github.com/appium/appium/pull/13708
## Integration tests
- [X] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

### CODE REVIEW

**Comment by akinsolb** [comment]
@manoj9788 Sorry for the delay, environment issues as usual

**Comment by akinsolb** [comment]
@manoj9788 I was thinking of restructuring all test namespace to match the command categories in appium.io documentation.

File: test/integration/IOS/Device/AppTests.cs [hunk 1]
```diff
@@ -9,14 +9,14 @@
 
 namespace Appium.Net.Integration.Tests.IOS.Device.App
 {
-    internal class ActivateAppTest
+    internal class AppTests
```
Comment (manoj9788) [inline test/integration/IOS/Device/AppTests.cs]: Makes sense.

**Comment by manoj9788** [comment]
Cheers! that makes sense and I was hinting at that. I'll add a few more tests. Thanks for restructuring this.

**Pull request merged**